gcs/CONFIGURATION.md

```diff
@@ -153,18 +153,13 @@ provider.
     The implementation of the AccessTokenProvider interface used for GCS
     Connector.
     
-*   `fs.gs.should.authenticate.per.request` (default: `false`)
+*   `fs.gs.refresh.access-token.per.request.enable` (default: `false`)
```

Comment:
Please also mention that "when this property is set to true, low level HTTP retries will be disabled".